scripts/tests/test_install_pi_transaction.py

```
@@ -262,16 +277,16 @@ def test_symlinked_pi_is_rejected_without_touching_link_or_external_tree(self):
             with self.subTest(command=command[-1]),tempfile.TemporaryDirectory() as d,tempfile.TemporaryDirectory() as external_d:
                 home=Path(d);external=Path(external_d);nested=external/'nested';nested.mkdir();payload=nested/'payload.bin';payload.write_bytes(b'\x00external\xffcontent\n');os.chmod(external,0o751);os.chmod(nested,0o753);os.chmod(payload,0o640);stamp=1_577_934_245_123_456_789
                 for index,target in enumerate((external,nested,payload)): os.utime(target,ns=(stamp+index,stamp+index+10))
-                link=home/'.pi';link.symlink_to(external,target_is_directory=True);before_link=self.path_snapshot(link);before_external=self.path_snapshot(external);env={**os.environ,'HOME':d}
+                link=home/'.pi';link.symlink_to(external,target_is_directory=True);before_link=self.path_snapshot(link);before_external=self.path_snapshot(external);env={**os.environ,'HOME':d};receipt=home/'failure-receipt.json'
```

Ruff (0.16.0)

[error] Multiple statements on one line (semicolon)

(E702)